### DISCUSSION TITLE
How does generative AI work anyway? A quick introduction to the fundamentals!

### DISCUSSION THREAD

**Discussion by cassiebreviu** [community/community#63185] [Programming Help]
At this point everyone has heard of, played with and/or experienced the power of generative AI in some way. How does it work though? In this short _Fundamentals of Generative AI_ module, you can walk through high-level how they work! This is a very approachable module and doesn't require a deep understanding of AI. Get started here 🏫👉https://learn.microsoft.com/training/modules/fundamentals-generative-ai/ and don't forget to post questions below 👇

**Comment by RamtinKosari** [comment]
Generative AI is a powerful technology that many people have become familiar with. If you're curious about how it works, the Fundamentals of Generative AI module provided by Microsoft Learn is a great resource. This module offers a high-level explanation of generative AI without requiring a deep understanding of AI concepts. It provides an approachable introduction to the topic.

In the module, you will explore how large language models (LLMs) enable AI applications and services to generate original content based on natural language input. You will learn about the role of generative AI in the development of artificial intelligence and how large language models play a crucial role in intelligent applications. The module also covers how Azure OpenAI supports the creation of intelligent applications and provides examples of AI-powered copilots that can assist humans in creative tasks.

By completing this module, you will achieve the following learning objectives:

Understand the significance of generative AI in the field of artificial intelligence.
Gain insights into large language models and their role in intelligent applications.
Learn about how Azure OpenAI facilitates the creation of intelligent applications.
Explore examples of copilots and understand the concept of good prompts.
To get started with the Fundamentals of Generative AI module, you can access it through the following link: [Fundamentals of Generative AI Module ↗](https://learn.microsoft.com/training/modules/fundamentals-generative-ai/). Feel free to explore the module and don't hesitate to ask any questions you may have by posting them below the module.

Enjoy your learning journey into the world of generative AI!
